docs/examples/101/sql-database/main.bicep

```diff
@@ -0,0 +1,47 @@
+param serverName string {
+  metadata: {
+    description: 'The name of the SQL logical server.'
+  }
+  default: uniqueString('sql', resourceGroup().id)
+}
+param sqlDBName string {
+  metadata: {
+    description: 'The name of the SQL Database.'
+  }
+  default: 'SampleDB'
+}
+param location string {
+  metadata: {
+    description: 'Location for all resources.'
+  }
+  default: resourceGroup().location
+}
+param administratorLogin string {
+  metadata: {
+    description: 'The administrator username of the SQL logical server.'
+  }
+}
+param administratorLoginPassword string {
+  metadata: {
+    description: 'The administrator password of the SQL logical server.'
+  }
+  secure: true
+}
+
+resource serverName_resource 'Microsoft.Sql/servers@2019-06-01-preview' = {
+  name: serverName
+  location: location
+  properties: {
+    administratorLogin: administratorLogin
+    administratorLoginPassword: administratorLoginPassword
+  }
+}
+
+resource serverName_sqlDBName 'Microsoft.Sql/servers/databases@2020-08-01-preview' = {
+  name: '${serverName_resource.name}/${sqlDBName}'
+  location: location
+  sku: {
+    name: 'Standard'
+    tier: 'Standard'
+  }
+}
```

Comment:
you'll need a newline at the end of the file in order to get the tests to pass. You can run the tests locally if you have the .NET 5 SDK installed and running `dotnet test`

Reply:
When you use the format command of VS Code it removes the last line

Reply:
I'm not crazy! Opened up #997 as I had the same issue

Reply:
Checking this option in VSCode solves the issue.
![Capture d’écran 2020-11-27 à 14 03 56](https://user-images.githubusercontent.com/5141735/100452527-9dd50b00-30b9-11eb-8b0f-cf0f7bef4a0a.png)